tests/app/main/test_services.py

```diff
@@ -823,6 +886,16 @@ def test_should_fail_if_incorrect_framework(
 @mock.patch('app.main.views.services.data_api_client', autospec=True)
 class TestSupplierEditUpdateServiceSection(BaseApplicationTest):
 
+    def _get_framework_response(self, **kwargs):
+        framework = {
+            "framework": "g-cloud",
+            "name": "G-Cloud 6",
+            "slug": "g-cloud-6",
+            "status": "live",
```

Comment:
You could use fun new Python 3 syntax here:

```
framework = {"framework": "g-cloud", "name": "G-Cloud 6", "slug": "g-cloud-6", "status": "live", **kwargs}
            
```

Reply:
Will a status in `kwargs` there overwrite the earlier `status`?

Reply:
Yep! It follows left-to-right order, so:
`{a: 1, **{a: 2}} == {a: 2}`
and
`{**{a: 2}, a: 1} == {a: 1}`

Reply:
Nice - it's neater like this - done.